pkgs/stdenv/generic/builder.sh

```diff
@@ -4,18 +4,20 @@ for i in $initialPath; do
     PATH=$PATH${PATH:+:}$i/bin
 done
 
-mkdir $out
+mkdir "$out"
```

Comment:
we don't need to quote every $out because it cannot contain spaces.

Reply:
Agreed, however IMO it's better to try to quote in every case you can, you don't lose anything by doing it.
I feel this is more like a "style guide" than a real error.
Just like using [ over [[. Sometimes you can, but overall having a single style would make readability easier I think.

Reply:
I'm happy to remove quotes where you think it's inappropriate. I have a different opinion, but I would like to focus on what we can all agree on. So if you let me know where you think I wrongly added quotes, I'll remove them.